lib/logstash/outputs/scalyr.rb

```diff
@@ -240,7 +240,9 @@ def register
     # Plugin level (either per batch or event level metrics). Other request
     # level metrics are handled by the HTTP Client class.
     @multi_receive_statistics = {
-      :total_multi_receive_secs => 0
+      :total_multi_receive_secs => 0,
+      :total_bignum_json_errors => 0,
+      :total_successful_json_conversions => 0
```

Comment:
Perhaps ``total_bignum_json_conversions`` would be a better name.

Reply:
To clarify, what should this be tracking? Successful JSON serialization, or successful calls to `convert_bignums`? Because at the moment `total_bignum_json_errors` tracks whenever we run into an issue with JSON serialization due to bignums, not failures in running `convert_bignums`.

Reply:
Ah I see your message below, all makes sense now